agrirouter-sdk-dotnet-standard-api/Service/Parameters/MessagePayloadParameters.cs

```diff
@@ -7,6 +7,17 @@ namespace Agrirouter.Api.Service.Parameters
     /// </summary>
     public class MessagePayloadParameters : Parameters
     {
+        /// <summary>
+        /// Every endpoint can send messages based on its capabilities. The size of a message is however limited. A message contains 2 parts: Header and Body. The limitation of a message is defined as follows:
+        /// - The maximum body size is equivalent of 1024000 characters/signs
+        /// - Since the chunking is performed on the raw message data this means, that we have to lower the MAX_LENGTH_FOR_MESSAGES to allow Base64 encoding afterwards.
+        /// - Total message size is limited to 1468000 characters/signs
+        /// - Messages that are above this limit will be rejected.
+        /// The AR will return an error indicating that the message size is above the limit.
+        /// If the message size is above 5 MB the AR will not return any error. In order to send messages with sizes above threshold, these messages must be split into chunks with the above limit.
+        /// </summary>
```

Comment:
This last sentence sounds like the real threshold is 5MB, which is not the case. Advise:
Split everything starting from "Total message size is limited to 14680000" as "Additional infos" and add the sentence "Do not try to sent more than MAX_LENGTH_FOR_MESSAGES of raw data in one message."

Reply:
Comment was copied from above

Reply:
Did not resolve this one, since the comment is the same as within the other SDK(s).